src/wp-admin/images/cp-logo-gradient.svg

```diff
@@ -0,0 +1 @@
9a7.55,7.55,0,0,1,2.17,5.72,7.56,7.56,0,0,1-2.17,5.73c-1.44,1.39-4.23,2.08-8.35,2.08h-5.07V34c0,.66-.3,1-.91,1Z" style="fill:#017f98"/><path d="M126.85,35a.72.72,0,0,1-.81-.82V16.71a.73.73,0,0,1,.82-.82h1.53a.73.73,0,0,1,.82.82V18.9a6.94,6.94,0,0,1,2.08-2.48,5.06,5.06,0,0,1,3-.84,4.9,4.9,0,0,1,1.66.2.42.42,0,0,1,.3.55l-.42,1.49c-.06.22-.21.31-.47.26a5.8,5.8,0,0,0-1.21-.13,4.21,4.21,0,0,0-3.53,1.92,8.76,8.76,0,0,0-1.39,5.24v9.08a.72.72,0,0,1-.81.82Z" style="fill:#017f98"/><path d="M139.78,24.3h10.29a8.65,8.65,0,0,0-1.31-4.47,4.06,4.06,0,0,0-3.64-1.92,5,5,0,0,0-3.59,1.41c-1,.95-1.56,2.6-1.75,5m-3.31,1.15a10.21,10.21,0,0,1,2.26-7,8.12,8.12,0,0,1,6.46-2.58,7.79,7.79,0,0,1,5.8,2.36c1.53,1.58,2.3,4.11,2.3,7.57,0,.53-.25.79-.75.79H139.78q.32,3.4,1.91,4.74a5.48,5.48,0,0,0,3.61,1.34,6.37,6.37,0,0,0,3.27-.69,6.22,6.22,0,0,0,1.66-1.36c.39-.49.77-.59,1.13-.28l1.08.9c.28.23.28.56,0,1a8.88,8.88,0,0,1-2,1.63A9.23,9.23,0,0,1,145.34,35c-3,0-5.27-.87-6.71-2.59a10.54,10.54,0,0,1-2.16-7" style="fill:#017f98"/><path d="M155.07,33c-.37-.34-.45-.66-.23-.95l.82-1.11c.26-.35.66-.33,1.18.05a9.42,9.42,0,0,0,2.39,1.27,8.82,8.82,0,0,0,3,.48,6.08,6.08,0,0,0,1.78-.24,3,3,0,0,0,1.37-.89,2.47,2.47,0,0,0,.6-1.74,2.59,2.59,0,0,0-.29-1.2,2.12,2.12,0,0,0-.87-.91,6.32,6.32,0,0,0-1.16-.54,18.42,18.42,0,0,0-1.84-.56c-.62-.14-1.42-.37-2.42-.68a8,8,0,0,1-1.89-.85A5.54,5.54,0,0,1,156,23.6a4.53,4.53,0,0,1-.66-2.62A4.35,4.35,0,0,1,156.38,18a5.32,5.32,0,0,1,2.7-1.68,11.36,11.36,0,0,1,3.15-.45,10.71,10.71,0,0,1,3.63.5,10.37,10.37,0,0,1,2.38,1.19.57.57,0,0,1,.16.93l-.82,1.06c-.25.34-.65.33-1.18,0a8.08,8.08,0,0,0-1.91-.86,7.17,7.17,0,0,0-2.17-.41,7,7,0,0,0-2,.25,2.74,2.74,0,0,0-1.33.85,2.08,2.08,0,0,0-.51,1.4,2.88,2.88,0,0,0,.2,1.1,2.21,2.21,0,0,0,.75.89,4.21,4.21,0,0,0,1.53.66l2.79.7a14,14,0,0,1,2.29.77,7,7,0,0,1,2.05,1.32,4,4,0,0,1,1,1.56,6.06,6.06,0,0,1,.27,1.86,4.63,4.63,0,0,1-1.15,3.19,5.55,5.55,0,0,1-2.58,1.73,11.79,11.79,0,0,1-3.42.44,12.88,12.88,0,0,1-4.65-.75A8.85,8.85,0,0,1,155.07,33" style="fill:#017f98"/><path d="M170.58,33c-.38-.34-.45-.66-.24-.95l.82-1.11c.27-.35.67-.33,1.19.05a9.25,9.25,0,0,0,2.39,1.27,8.76,8.76,0,0,0,3,.48,6.11,6.11,0,0,0,1.79-.24,3,3,0,0,0,1.37-.89,2.46,2.46,0,0,0,.59-1.74,2.58,2.58,0,0,0-.28-1.2,2.17,2.17,0,0,0-.87-.91,6.64,6.64,0,0,0-1.16-.54,18.6,18.6,0,0,0-1.85-.56c-.61-.14-1.41-.37-2.41-.68A7.66,7.66,0,0,1,173,25.1a5.29,5.29,0,0,1-1.56-1.5,4.53,4.53,0,0,1-.66-2.62A4.3,4.3,0,0,1,171.89,18a5.32,5.32,0,0,1,2.7-1.68,11.34,11.34,0,0,1,3.14-.45,10.74,10.74,0,0,1,3.64.5,10.29,10.29,0,0,1,2.37,1.19.57.57,0,0,1,.17.93l-.82,1.06c-.26.34-.65.33-1.19,0a7.72,7.72,0,0,0-1.91-.86,6.69,6.69,0,0,0-4.13-.16,2.69,2.69,0,0,0-1.33.85,2.08,2.08,0,0,0-.51,1.4,2.88,2.88,0,0,0,.2,1.1,2.06,2.06,0,0,0,.75.89,4.21,4.21,0,0,0,1.53.66l2.79.7a14.28,14.28,0,0,1,2.29.77,6.92,6.92,0,0,1,2,1.32,4.18,4.18,0,0,1,1,1.56,6.06,6.06,0,0,1,.27,1.86,4.64,4.64,0,0,1-1.16,3.19,5.57,5.57,0,0,1-2.57,1.73,11.81,11.81,0,0,1-3.43.44,13,13,0,0,1-4.65-.75,8.9,8.9,0,0,1-2.5-1.28" style="fill:#017f98"/></svg>
```

Comment:
Please use this SVG file directly and remove the `.png` added here (it's very low-resolution anyway).  Fortunately, we are past the point where we need to care about browsers which can't handle SVG images, and we've already added several others.

Reply:
I've updated the style sheet to use the direct URL for the logo image, but am unable to delete the 2 images from the pull request...I get a non-working delete button for the svg, and no delete button for the png.

Reply:
The GitHub web UI doesn't support this, I guess.  I can take care of it.

Reply:
Thanks. I'll let you mark this as resolved when ready.

Reply:
Done.  Note, I updated the stylesheet to use the copy of the SVG from the core code instead of a remote URL.